Documentation.md

```diff
@@ -16,6 +16,10 @@ Click [here](Architecture.md) to view a quick architecture overview of how the
 Amazon Connect Streams API works.
 
 ## Getting Started
+
+### Upgrading to the OmniChannel CCP (AKA CCPv2)?
+If you are migrating to the new CCP, we encourage you to upgrade to the latest version of this repository. You should also upgrade to [the latest version of RTC-JS](https://github.com/aws/connect-rtc-js) as well, if you are using it. For a complete migration guide to the new CCP, and to fully understand the differences when using Streams with the new CCP, please see this post: https://docs.aws.amazon.com/connect/latest/adminguide/upgrade-to-latest-ccp.html
+
 ### Whitelisting
 The first step to using the Streams API is to whitelist the pages you wish to embed.
 For our customer's security, we require that all domains which embed the CCP for
```

Comment:
Is this a complete sentence? I think I'm missing the second part in my view. "all domains which embed the CCP for.
Can we link to: https://github.com/amazon-connect/amazon-connect-streams#whitelisting

Reply:
Click the down arrow and you will see the rest :)